test/e2e/e2e.go

```diff
@@ -252,6 +257,14 @@ func createClusterExternalContainer(containerName string, containerImage string,
 	return strings.Trim(ip, "\n")
 }
 
+func ipv6AddressForContainer(containerName string) string {
```

Comment:
TODO, not for doing it here, but we should consolidate this as with current helpers in unit, just return both IPs ordered sounds ok to me

Reply:
Let me do it here